shell/components/ExplorerProjectsNamespaces.vue

```diff
@@ -96,6 +96,7 @@ export default {
         STATE,
         NAME,
         this.groupPreference === 'none' ? project : null,
+        NS_SNAPSHOT_QUOTA,

```

Comment:
The new column will be visible also in Rancher after porting the changes in the rancher/shell. I think we should display the column only if `isHarvester` === true; we have other examples in the code.

Reply:
Modified in a460f88bf124cc82f3d8f025dbc30cef94c77b5d.